CITATION.cff

```diff
@@ -0,0 +1,36 @@
+# following https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+authors:
+- family-names: "Ramesh Babu"
+  given-names: "Reshma"
+  orcid: "http://orcid.org/0000-0002-2517-5347"
+cff-version: 1.2.0
+date-released: "2023-12-04"
+identifiers:
+  - description: "The GitHub release URL of tag 1.0.0."
+    type: url
+    value: "https://github.com/Imageomics/Butterfly-mimicry/releases/tag/v1.0.0"
+  - description: "The GitHub URL of the commit tagged with 1.0.0."
+    type: url
+    value: "https://github.com/Imageomics/Butterfly-mimicry/tree/<commit>"
+keywords:
+  - "imageomics"
+  - "butterflies"
+  - "mimicry"
+  - "machine learning"
+  - "computer vision"
+  - "traits"
+license: MIT
+message: "If you use this software, please cite both the thesis from preferred-citation and the software itself."
```

Comment:
Do you know that the preferred citation will be shown as "preferred-citation"? Perhaps we will only know once merged.

Reply:
So it doesn't seem like it's using the `message` in the display, am I right?

Reply:
Yes, it doesn't seem so, and I'm not sure why, as that is the purpose of the [`message`](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#message) parameter.